setup.cfg

```diff
@@ -38,6 +37,17 @@ dev =
     orquestra-opt[cma]
     orquestra-cirq[qsim]
 
+qulacs =
```

Comment:
You should also add the new installations to `dev`.
I'm not sure what the syntax should be, but I think it was something like `.[qiskit]`

Reply:
Done